ptsprojects/zephyr/gap.py

```diff
@@ -229,6 +229,12 @@ def test_cases(pts):
         ZTestCase("GAP", "GAP/BROB/OBSV/BV-02-C",
                   cmds=pre_conditions,
                   generic_wid_hdl=gap_wid_hdl),
+        ZTestCase("GAP", "GAP/BROB/OBSV/BV-03-C",
```

Comment:
Those test cases are available on PTS workspace while not in TestPlan. so I would omit those. Do you see those in your test plan?

Reply:
It is enabled in my TestPlan due to the supported spec is 5.1.
If we choose to use 5.0, these will be disabled.